app/views/students/shared/_student_form.html.erb

```diff
@@ -27,6 +27,32 @@
       <%= render '/students/shared/student_particulars', f: f %>
       <hr>
 
+      <h4 class="card-title">Attachments</h4>
+      <div id="student-attachments">
+      <div class="form-group row">
+        <table class="table">
+          <thead>
+          <tr>
+            <th><%= f.label :type, 'Type' %></th>
+            <th><%= f.label :filename, 'Filename' %></th>
+            <th><%= f.label :notes, 'Notes' %></th>
+            <th></th>
+          </tr>
+          </thead>
+          <tbody class="attachments">
+          <%= f.fields_for :attachments do |attachment| %>
+              <%= render '/students/shared/attachments', f: attachment %>

```

Comment:
Might not better if we show the download link here instead of the form field. Once the file is uploaded successfully, you wouldn't need to upload / replace it again.